File: dayTwo/python/After Break/father_and_son_age.py
def calculateTheNumberOfYears(fatherAge, sonAge) :
    twiceSonAge = sonAge * 2
    difference = fatherAge - twiceSonAge
    
    if difference < 0 :
        difference = difference * -1
        print(f"{difference} years ago")
    
    elif difference == 0:
        print("It will be this year")
    else:
        print(f"{difference} years from now")
    
            
print(calculateTheNumberOfYears(40, 18))

File: dayTwo/javaScript/After Break/fatherAndSonAge.js
function calculateTheNumberOfYears(fatherAge, sonAge){
    let twiceSonAge = sonAge * 2;

    let difference = fatherAge - twiceSonAge;

    if(difference < 0){
        difference = difference * -1;
        console.log(`${difference} years ago`)       
    }
    else if(difference == 0){
        console.log("It will be this year!!")
    }
    else{
        console.log(`${difference} years from now`)
    }

}

console.log(calculateTheNumberOfYears(60, 28)) 

